examples/Tables/Table.md

```diff
@@ -176,7 +176,7 @@ import {
 
 ### More Complex Layouts
 
-Some tables may require more complex layouts, such as cells that span multiple columns/rows or layered, hierarchical headers. For example:
+Some tables may require more complex layouts, such as cells that span multiple columns/rows, or layered headers. For example:
 
 ```tsx
```

Comment:
Is this supposed to be `tsx` or should it be `jsx`? The table wasn't showing up for me in the markdown until I changed it to `jsx`

Reply:
I think you need to be on the newest version of `styleguidist` for `tsx` to work (we bumped the version up in commit